resources/languages/ko.json

```diff
@@ -143,7 +143,7 @@
 			"max-reverts": "저장할 실행 취소/다시 실행 보관의 최대량입니다. 이전 것들은 자동으로 비워집니다",
 			"reset-decrement-brush": "초기 실행 크기로 감소 브러시 자동으로 초기화",
 			"save-brush": "서버 다시 시작 시에 플레이어의 브러시 속성 저장",
-			"drop-plants": "낙엽청소기 유형으로 파괴된/불어진 식물 아이템 떨어뜨리기",
+			"drop-plants": "낙엽청소기 유형으로 파괴된/청소된 식물 아이템 떨어뜨리기",
```

Comment:
While I still think it can be translated to blow, I thought it was better to translate it to clean because of what I said in the previous comments - please tell me if you don't like this, so that I can revert this.

Reply:
That should be fine. As long as it's understandable 👍